projects/demo/src/modules/app/app.routes.ts

```diff
@@ -229,20 +229,16 @@ export const ROUTES: Routes = [
         title: 'Label ',
         loadComponent: async () => import('../experimental/label'),
     }),
+    route({
+        path: DemoRoute.Surface,
+        title: 'Surface ',
+        loadComponent: async () => import('../components/surface'),
+    }),
```

Comment:
what are this changes? besides Calendar, have any other components been switched to standalone?

try to make one pull request per component